src/Microsoft.AspNetCore.Razor.Language/RazorParserVersion.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    public enum RazorParserVersion
+    {
+        Version1_0,
+        Version1_1,
+        Version2_0,
+        Version2_1,
```

Comment:
Will there be a need to add more here other than for every major release? Eg. We'll need different parser behavior for patch releases?

Reply:
No, not for patches

Reply:
We might want to do these the way Roslyn does - https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/LanguageVersion.cs